src/wpilib/robot-states.md

```diff
@@ -0,0 +1,36 @@
+# Robot States
+
+Robots programmed with WPILib can be in one of four states:
+
+- **Disabled**:
+  The robot does not accept user input and shouldn't move mechanisms.
+  It is safe for humans to be around the robot while it is disabled.
+  This is the default state.
+- **Teleoperated**:
+  The robot accepts user input and moves mechanisms.
+  This is the most common state used while testing,
+  and the teleoperated period of the FRC game is spent in this state.
+- **Autonomous**:
+  The robot does not accept user input, and instead goes through a sequence of preprogrammed actions.
+  This state is active for the first 15 seconds in an FRC match.
+- **Emergency Stopped (E-Stopped)**:
+  This is a special state which is identical to the disabled state,
+  except that the bot cannot be re-enabled after it has been emergency stopped.
+
+Here's a table comparing each state:
+
+| State          | Moves | Accepts Input | Can Be Enabled |
+| -------------- | ----- | ------------- | -------------- |
+| Disabled       | No    | No            | Yes            |
+| Teleoperated   | Yes   | Yes           | N/A            |
+| Autonomous     | Yes   | No            | N/A            |
+| Emergency Stop | No    | No            | No             |
```

Comment:
IMO consider making this "Emergency Stop (Disabled)"

Reply:
Why?

Reply:
Effectively the robot's behavior is the same during an emergency stop, I don't think not being able to remove it from that state is enough to warrant a different section